runelite-client/src/main/java/net/runelite/client/plugins/dropparty/DropPartyPlugin.java

```diff
@@ -138,8 +138,9 @@ private void onGameTick(GameTick event)
 			{
 				continue;
 			}
-			if (Text.standardize(player.getName()).equalsIgnoreCase(playerName))
+			if (player.getName().equalsIgnoreCase(playerName))

```

Comment:
Rather than this maybe `Text.sanitize` would be better here

Reply:
Do you also need to refind the player every game tick? Seems to me like you should only have to find it once and can then keep track of them until they've despawned

Reply:
that was for handling if the player is offscreen it still ticks down correctly, could be possible to do it without redefining every tick